Twig/DoctrineExtension.php

```diff
@@ -39,7 +39,8 @@ public function getFilters()
         return array(
             new \Twig_SimpleFilter('doctrine_minify_query', array($this, 'minifyQuery')),
             new \Twig_SimpleFilter('doctrine_pretty_query', 'SqlFormatter::format', array('is_safe' => array('html'))),
```

Comment:
This filter should be deprecated in favor of ``doctrine_highlight_sql``, or the existing filter name should be kept.